chart/values.yaml

```diff
@@ -18,11 +18,18 @@ secret:
 configMap:
   responseConfig: |-
     {
-       "Versions": [{
-          "Name": "v1.0.0",
-          "ReleaseDate": "2020-05-18T12:30:00Z",
-          "Tags": ["latest"]
-       }]
+      "versions": [{
+        "name": "v1.0.0",
+        "releaseDate": "2020-05-18T12:30:00Z",
+        "tags": ["latest"]
+      }]
+    }
```

Comment:
Would the response format change to impact the existing installations to get the version info?

Reply:
Thank you for the catch!

It still works when the client is Golang because [Json.Unmarshal() function does case-insensitive match](https://pkg.go.dev/encoding/json#Unmarshal:~:text=Unmarshal%20matches%20incoming%20object%20keys%20to%20the%20keys%20used%20by%20Marshal%20(either%20the%20struct%20field%20name%20or%20its%20tag)%2C%20preferring%20an%20exact%20match%20but%20also%20accepting%20a%20case%2Dinsensitive%20match). For example: https://go.dev/play/p/8G8m0rfh_20

I am not sure if the client are written by another language. Perhaps I will have to rollback this one to be safe.

Reply:
I think it should be fine for us.